publish/publish.proj

```diff
@@ -262,39 +262,32 @@
   </Target>
 
   <Target Name="DoPushCoreHostPackagesToAzure"
-          DependsOnTargets="PublishToAzureBlobFeed;PublishSymbolsToAzureBlobFeed"
+          DependsOnTargets="PublishToAzureBlobFeed"
           Condition="'$(OfficialPublish)' == 'true'"/>
 
   <Target Name="PublishToAzureBlobFeed">
     <Error Condition="'$(PackagesUrl)'==''" Text="Missing property PackagesUrl" />
     <Error Condition="'$(TransportFeedAccessToken)'==''" Text="Missing property TransportFeedAccessToken" />
     <PropertyGroup>
       <ExpectedFeedUrl>$(PackagesUrl)</ExpectedFeedUrl>
+      <AssetManifestFilePath>$(BUILD_SOURCESDIRECTORY)/artifacts/log/AssetManifest/Manifest.xml</AssetManifestFilePath>
     </PropertyGroup>
     <ItemGroup>
       <ItemsToPush Remove="@(ItemsToPush)" />
       <ItemsToPush Include="@(_DownloadedStandardPackages)"/>
+       <ItemsToPush Include="@(_DownloadedSymbolsPackages)"/>
     </ItemGroup>
     <PushToBlobFeed ExpectedFeedUrl="$(ExpectedFeedUrl)"
                     AccountKey="$(TransportFeedAccessToken)"
```

Comment:
Publishes only nuget packages , what about other assets like native installers, zips, checksums ? Is it required ?

Reply:
@mmitche  , this is the same question I have here . I will look what it takes to includes all these missing assets. BTW, do we need checksums too ?

Reply:
Yes. I think we'd even create the container if it does not exist